--- w3m-0.5.2/w3mbookmark.c	2007-05-30 18:19:50.000000000 -0700
+++ w3m-0.5.2/w3mbookmark.c.new	2008-12-22 23:22:27.000000000 -0800
@@ -99,7 +99,7 @@
 	fprintf(f, "<body>\n<h1>Bookmarks</h1>\n");
 	fprintf(f, "<h2>%s</h2>\n<ul>\n", section);
 	fprintf(f, "<li><a href=\"%s\">%s</a>\n", url, title);
-	fprintf(f, end_section);
+	fprintf(f, "%s", end_section);
 	fprintf(f, "</ul>\n</body>\n</html>\n");
 	fclose(f);
     }
